src/tools/clippy/.github/driver.sh

```diff
@@ -28,14 +28,14 @@ diff normalized.stderr tests/ui/cstring.stderr
 
 
 # make sure "clippy-driver --rustc --arg" and "rustc --arg" behave the same
-SYSROOT=`rustc --print sysroot`
+SYSROOT=$(rustc --print sysroot)
 diff <(LD_LIBRARY_PATH=${SYSROOT}/lib ./target/debug/clippy-driver --rustc --version --verbose) <(rustc --version --verbose)
 
 
 echo "fn main() {}" > target/driver_test.rs
 # we can't run 2 rustcs on the same file at the same time
-CLIPPY=`LD_LIBRARY_PATH=${SYSROOT}/lib ./target/debug/clippy-driver ./target/driver_test.rs --rustc`
-RUSTC=`rustc ./target/driver_test.rs`
+CLIPPY=$(LD_LIBRARY_PATH=${SYSROOT}/lib ./target/debug/clippy-driver ./target/driver_test.rs --rustc)
+RUSTC=$(rustc ./target/driver_test.rs)
 diff <($CLIPPY) <($RUSTC)
```

Comment:
This ... does not look right. Also this is a change to submodule so I'm not sure you should make it here, but clippy uses `subtree` so it might be special.

Reply:
Is there any difference between `` and $()? I changed it because shellcheck warned about it and as far as I know they do the same thing.

Reply:
Oh no your change is fine, I mean running eval on dynamically generated code. Normally I'd suggest to add quotes around the `$(...)`  but since I don't understand this code I'm afraid to change it.